contrib/helm/harbor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: harbor
-version: 0.1.2
-appVersion: 1.4.0
+version: 0.1.3
```

Comment:
How about bump the version to 0.2.0 ?

Reply:
How about bump the version to 0.2.0 ?  It should be at least a minor version bump.

Reply:
ok